cmd/server/main.go

```diff
@@ -112,12 +112,18 @@ func main() {
 		sub.Use(middleware.RequireAuth(ctx, auth, db, config.SessionCookieDuration).Handle)
 		sub.Handle("", home.New(ctx, config, db, renderHTML)).Methods("GET")
 
-		// API for creating new verification codes. Called via AJAX.
-		sub.Handle("/issue", issueapi.New(ctx, config, db)).Methods("POST")
-
 		// API for obtaining a CSRF token before calling /issue
 		// Installed in this context, it requires authentication.
 		sub.Handle("/csrf", csrfctl.NewCSRFAPI()).Methods("GET")
+
+		issueSub := sub.PathPrefix("/issue").Subrouter()
+		// TODO(crwilcox): add these to the config.
```

Comment:
wanted to get agreement on approach. Can expand this pr to config once we reach concensus.